apps/redi-connect/src/pages/app/applications/MobileView.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import Select from 'react-select'
```

Comment:
Brilliant!!! So simple, and exactly what we needed. I totally forgot we use this package, and I had no clue that it's actually used by `<FormSelect>`

Reply:
@helloanil, check it out :) Nice surprise from @katamatata